.github/workflows/unit_tests.yml

```diff
@@ -54,4 +54,4 @@ jobs:
           pip install -r requirements/tests.txt
       - name: Run unittests
         run: |
-          pytest test/unittests
+          pytest --cov-config pyproject.toml --cov-report html --cov-report term --durations 5 --cov=ovos_dinkum_listener
```

Comment:
Consider a pytest.ini file, as this could start to get unwieldy

Reply:
Or add it to pyproject.toml, which I just noticed you added

Reply:
These are extra options on top of the ones in pyproject.toml. Adding coverage is not free, and often a single run of `pytest` is enough for a developer, so I wanted to ensure the pyproject-level options enabled all test by default, but I only wanted the extra coverage reporting on the CI.

But this is a design decision, and making CI and the user the same by default is reasonable. Typically what I do is I add a "./[run_tests.sh](https://github.com/Erotemic/ubelt/blob/main/run_tests.py)" script to my repo root, which does the tests with coverage, but leave pytest just run the tests and don't bother with coverage.

Reply:
Actually this is a better example of a more modern run_tests.py that I use:

https://gitlab.kitware.com/computer-vision/kwimage/-/blob/main/run_tests.py?ref_type=heads